docs/02-app/01-building-your-application/04-caching/index.mdx

```diff
@@ -0,0 +1,421 @@
+---
+title: Caching in Next.js
+nav_title: Caching
+description: An overview of caching mechanisms in Next.js.
+---
+
+Next.js has a few caching mechanisms to improve your application's performance. This page provides an in-depth look at caching, the APIs you can use to configure them, and how they interact with each other.
+
+## Overview
+
+{/* TO DO: Diagram showing how caching mechanisms come into play in the request lifecycle, going from initial request, to server, to response, to router cache, to subsequent navigation with new request. */}
+
+Here's a high-level overview of the different caching mechanisms in Next.js and their purpose:
+
+| Mechanism                     | What is cached?                | Where is it cached? | Why is it cached?                                             | Duration                        |
+| ----------------------------- | ------------------------------ | ------------------- | ------------------------------------------------------------- | ------------------------------- |
+| [React Cache](#react-cache)   | Return values of functions     | Server              | Reduce data store calls and share data between route segments | Per-request lifecycle           |
+| [Data Cache](#data-cache)     | Return values of data requests | Server              | Store data across user requests and deployments               | Persistent (can be revalidated) |
+| [Render Cache](#render-cache) | Rendered React Components      | Server              | Reduce rendering cost and improve performance                 | Persistent (can be revalidated) |
```

Comment:
Let's use "Route Cache" for this concept, it reflects the reality better.

Under `What is cached` we can say `Rendered HTML and RSC payload`